src/client/app/reducers/admin.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as adminActions from '../actions/admin';
+import { chartTypes } from './graph';
+
+const defaultState = {
+	displayTitle: '',
+	defaultChartToRender: chartTypes.line,
+	defaultBarStacking: false,
+	isFetching: false
+};
```

Comment:
I think it would be better to track the current settings and the settings being edited separately, but I don't think it's absolutely necessary.
It is necessary, however, to have a flag to store whether or not there are any changes that have yet to be submitted.

Reply:
Why is it important to check if there are any changes that have yet to be submitted?

Reply:
Because if there are no changes that need to be submitted, then it should not be possible to send the network requests. It also rules out somehow sending a second request while the first is still in flight. It's also generally good practice to write redux actions defensively --- you never know how they'll be used in the future.